gatherling/report.php

```diff
@@ -292,3 +281,20 @@ function print_verify_resultForm($report, $match_id, $player, $drop, $opponent,
     echo "<div class=\"clear\"> </div>\n";
 }
 ?>
+
+<script>
+function verify_input(form) {
+    //Check if the player wants to drop
+    if(form[9].checked) {
+        return confirm('Are you sure you want to drop? This cannot be undone.');
+    }
+
+    //Check if no result being selected
+    if(!form[5].checked && !form[6].checked && !form[7].checked && !form[8].checked) {
```

Comment:
I don't like using numerical indices here,  It's confusing and could cause issues with a later refactor.